.github/actions/docker-image/action.yml

```diff
@@ -0,0 +1,41 @@
+name: Espresso Docker Image
```

Comment:
i just realized since the service names are standardized now we could matrix with one value per service then we don't need this action